src/modal/modal.js

```diff
@@ -80,7 +80,8 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.transition'])
       restrict: 'EA',
       scope: {
         index: '@',
-        animate: '='
+        animate: '=',
+        windowScope: '=?'
```

Comment:
This can probably be just `=` since we don't have a use of modal-window that doesn't declare the `window-scope` attribute. Even if windowScope is null, it'll still be bound fine.